ui/app/styles/_code.scss

```diff
@@ -45,9 +45,12 @@
     }
 
     .icon {
+      display: flex;
+      align-items: center;
+      justify-content: center;
       width: 12px;
       height: 12px;
-      font-size: scale.$base;
+      font-size: 12px;
```

Comment:
not just related to this PR, but I'd encourage using `rem` units or size variables from the `scale` module instead of `px`

Reply:
I went with the 12px because the `font-size` of `copyable-code` was also 12px, but will definitely try to do that more!